qa/vagrant/build.gradle

```diff
@@ -64,7 +64,8 @@ String upgradeFromVersion
 
 String maybeTestsSeed = System.getProperty("tests.seed", null);
 if (maybeTestsSeed != null) {
-  seed = new BigInteger(maybeTestsSeed, 16).longValue()
+  def (masterSeed, testSeed) = maybeTestsSeed.tokenize(':')
+  seed = new BigInteger(masterSeed, masterSeed.size()).longValue()
```

Comment:
Oh! It should be 16 indeed. I misinterpreted the meaning of second parameter of the ctor. Will change back.